modules/qtwidgets/include/modules/qtwidgets/codeedit.h

```diff
@@ -80,9 +82,9 @@ class IVW_MODULE_QTWIDGETS_API CodeEdit : public QPlainTextEdit {
     void updateLineNumberArea(const QRect&, int);
 
     QWidget* lineNumberArea_;
-    std::vector<std::shared_ptr<std::function<void()>>> callbacks_;
-    ivec4 textColor_;
-    ivec4 highLightColor_;
+    vec4 textColor_;
+    vec4 highLightColor_;
+    SyntaxHighligther* sh_;
```

Comment:
Type: Highighter
I guess the syntax highlighter is a Qt widget and therefore deletion is managed by Qt. Do we want to add this to the name to make it clear it is Qt?

Reply:
That is not a convention that we have previously used